hack/make-rules/test.sh

```diff
@@ -57,6 +57,10 @@ kube::test::find_dirs() {
         -o -path './test/e2e_node/system/*' \
       -name '*_test.go' -print0 | xargs -0n1 dirname | sed "s|^\./|${KUBE_GO_PACKAGE}/|" | LC_ALL=C sort -u
 
+    # run tests for client-go
+    find ./staging/src/k8s.io/client-go -name '*_test.go' \
```

Comment:
Does this mean we run a number of tests twice (until we have removed all code copying?). The unit test CI job takes long already now.